memopol/settings.py

```diff
@@ -220,9 +220,9 @@
 
 COMPRESS_PRECOMPILERS = (
     # ('text/coffeescript', 'coffee --compile --stdio'),
-    ('text/less', 'lessc {infile} {outfile}'),
-    ('text/x-sass', 'sass {infile} {outfile}'),
-    ('text/x-scss', 'sass --scss {infile} {outfile}'),
+    ('text/less', 'lesscpy {infile}'),
+    # ('text/x-sass', 'sass {infile} {outfile}'),
+    # ('text/x-scss', 'sass --scss {infile} {outfile}'),
```

Comment:
A number of python portage/binding for sass exist if neede (but I don't know if we use any sass)

https://pypi.python.org/pypi/sass
https://pypi.python.org/pypi/less2sass
https://pypi.python.org/pypi/django-libsass
https://pypi.python.org/pypi/isass